galaxy_api/api/ui/viewsets/namespace.py

```diff
@@ -61,8 +63,12 @@ def get_queryset(self):
 
 class MyNamespaceViewSet(NamespaceViewSet):
     def get_queryset(self):
-        # FIXME(cutwater): this view should aso return all the namespaces if
-        # the user is part of the partner engineering team. Don't know how
-        # we plan to handle that.
-        return models.Namespace.objects.filter(
+        # All namespaces for users in the partner-engineers groups
+        if self.request.user.groups.filter(name__in=settings.NAMESPACE_ADMIN_AUTH_GROUPS):
```

Comment:
It might be cleaner to use `is_partner_engineer = permissions.IsPartnerEngineer.has_permissions(self.request, self)`. That way if we change the definition of a partner engineer later on, the changes will get applied everywhere.

https://github.com/ansible/galaxy-api/blob/master/galaxy_api/api/permissions.py#L7

Reply:
Agreed, I like IsPartnerEngineer better.